com.github.birros.WebArchives.json

```diff
@@ -0,0 +1,275 @@
+{
+    "id": "com.github.birros.WebArchives",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "web-archives",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=x11",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--talk-name=org.freedesktop.Tracker1",
+        "--filesystem=home",
```

Comment:
Do you need all of home? Where would you expect files to be stored?

Reply:
As i said in the first message, as long as the print portal is not implemented in WebKitGTK+, write access to the home is required to save PDF files.